xla/backends/cpu/benchmarks/hlo_benchmark_runner.h

```diff
@@ -41,7 +41,8 @@ absl::Status RunHloBenchmark(benchmark::State& state,
                              absl::string_view hlo_module,
                              absl::Span<const Literal* const> args,
                              StrToStrMapping replacements = {},
-                             bool disable_parallel_task_assigner = false);
+                             bool disable_parallel_task_assigner = false,
+                             bool use_thunk_runtime = true);
```

Comment:
The function interface has changed. Parameters are now put in a struct `HloBenchmarkOptions`. Please resolve conflicts and add the `use_thunk_runtime` option to the struct instead.